src/main/java/clients/TemporaryClient.java

```diff
@@ -5,20 +5,20 @@
 /**
  * {@link TemporaryClient} in {@code Matchmade} consists of three things:
  * <ul>
- *     <li>{@code self data} - data which is describing client trying to find a match</li>
- *     <li>{@code searching data} - these are the values wanted in clients which will be a match for current client</li>
- *     <li>{@code client id} - internal unique identifier for a client</li>
+ * <li>{@code self data} - data which is describing client trying to find a match</li>
```

Comment:
😐

Reply:
what do you mean?

Reply:
I mean the lack of indentation for `<li>` tags withing `<ul>` tag is disturbing